docs/topics/arrays.md

```diff
@@ -320,28 +294,34 @@ fun main() {
 ### Convert to Map
 
 To convert an array to a `map`, use the [`toMap()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/to-map.html) function.
-Only an array of [`Pair`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-pair/) can be converted to a map: `Array<Pair<K,V>>`.
-The first value of a `Pair` instance becomes a key and the second becomes a value.
+Only an array of [`Pair<K,V>`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-pair/) can be converted to a map.
+The first value of a `Pair` instance becomes a key and the second becomes a value. In the example below, we use the [infix notation](functions.md#infix-notation) to call the [`to()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to.html) function to create tuples of `Pair`.
```

Comment:
remove the parentheses from `to()`, since it's pretty much exclusively called as infix